### ISSUE TITLE
Address invalid localization files

### ISSUE THREAD

**Issue by jonasraoni** [pkp/pkp-lib#8081]
**Describe the bug**
Some of our localization files might have an invalid syntax, they might work, due to lax parsers, but also cause unexpected issues with other softwares:
- Weblate might wipe out unrelated content: https://github.com/pkp/ops/commit/cc6187b39b624346d60ec9b97a901d90403ac0e1
- The `gettext` package might ignore the text: https://github.com/php-gettext/Gettext/issues/282

**Solution**
- Ensure the `.po` files are valid at build time.
- For `stable-3_3_0` the lax behavior should be kept at runtime to avoid breaking plugins.
- Fix the localization files

**What application are you using?**
OJS/main

**Comment by jonasraoni** [comment]
@asmecher I've created a PR for the `pkp-lib/main` here:
- https://github.com/pkp/pkp-lib/pull/8094

I've tried to separate the commits, except the 4f379aa08daa4a8703a5b1dcf4e763feceba864d, where I used the `gettext` package to setup the plurals, it formatted the entries and also removed duplicated entries inside the same file.

I've also added a generic validator for the locale files.

**Comment by jonasraoni** [comment]
For the `stable-3_3_0`, I can pass my modified validator through the locale files, and fix the breaking issues.

I also think it's ok to add the [validate-locale.sh](https://github.com/pkp/pkp-lib/pull/8094/commits/27741344eef60c2b3a38d01d827b18b3050f7eeb) in a non-breaking way (e.g. emit errors without failing the build)

**Comment by asmecher** [comment]
Please hold off on this until we've reached a decision about translation workflows on this discussion: https://github.com/pkp/pkp-lib/discussions/8038

**Comment by jonasraoni** [comment]
Ok! So I'll just commit fixes for the broken keys (e.g. missing quotes, bad escaping, ...).

**Comment by asmecher** [comment]
Yes please -- I've recently merged translations from Weblate so now is a good time.

**Comment by jonasraoni** [comment]
Just to confirm, I've updated the broken files.

**Comment by jonasraoni** [comment]
@asmecher I've reviewed and moved the fixes related to duplicated keys here https://github.com/pkp/pkp-lib/issues/8119
